apps/api/src/modules/library/books.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
+import {BookRepository} from "./repository/book.repository";
+import {BookResolver} from "./resolvers/book.resolver";
+import {BookServiceFactory} from "./resolvers/use-cases";
+import {PrismaService} from "../../core/prisma/prisma.service";
+
+@Module({
+    providers: [
+        BookRepository,
+        PrismaService,
+        BookResolver,
+        BookServiceFactory,
+    ],
+})
+export class BooksModule {}
```

Comment:
Should this be called `BooksModule` or `BookModule` or `LibraryModule`?
Naming consistency is crucial.
Take a look at the `modules` folder and see that every other module has the folder named the same way as the module itself.
Furthermore, the modules names are all singular, not plural, then it should be `BookModule` or `LibraryModule` (not book__s__ )